wikidict/parse.py

```diff
@@ -73,16 +67,13 @@ def process(filename: str, locale: str) -> Dict[str, str]:
         word, code = xml_parse_element(element)
         if not word or not code or ":" in word:
             continue
-
-        if any(head_section in code for head_section in head_sections[locale]):
-            words[word] = code
```

Comment:
About that change, I was wondering if we can keep all words. Because we may want to add/remove a section from the locale and still be able to reparse the whole data from `--render` instead of having to rerun `--parse` first.

FTR here is the diff on my machine:
```shell
$ time python -m wikidict fr --parse
>>> Processing data/fr/pages-20201201.xml ...
>>> Saved 1,823,776 words into data/fr/data_wikicode-20201201.json
>>> Parse done!
152,28s user 2,99s system 99% cpu 2:35,46 total

$ time python -m wikidict fr --parse
>>> Processing data/fr/pages-20201201.xml ...
>>> Saved 3,953,853 words into data/fr/data_wikicode-20201201.json
>>> Parse done!
166,52s user 7,96s system 99% cpu 2:55,53 total
```
So 30 seconds to get the whole data. But it may have more impact on other steps as the number of words to handle is twice.

Reply:
I have no strong opinion right now, WDYT?

Reply:
If we do this, it could be beneficial to call `if any(head_section in code for head_section in head_sections[locale]):` before parsing the wikicode with wtp, because we parse a lot of words we don't need.

Reply:
Yes, that would work. I will move the code then.